juju/juju-qa-jenkins
# [JUJU-1204] Enable test-network on Google and Azure clouds

This is to ensure these integration tests have equal or greater coverage
than the legacy nw-network tests

## QA Steps

Deploy to jenkins and verify the jobs are green
https://jenkins.juju.canonical.com/job/test-network-multijob/44
